src/otx/algo/keypoint_detection/utils/data_sample.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) OpenMMLab. All rights reserved.
+"""Implementation of pose data sample."""
+from __future__ import annotations
+
+from typing import Sequence, Union
+
+import numpy as np
+import torch
+from otx.algo.utils.mmengine_utils import InstanceData
+
+LABEL_TYPE = Union[torch.Tensor, np.ndarray, Sequence, int]
+
+
+class PoseDataSample(InstanceData):
```

Comment:
what is the rational behind for this decision?

Reply:
During mm decoupling, jaeguk temporarily brought `InstanceData` and we decided to remove it to use more simple data structure.
At that time, you also agreed with this :)

Reply:
sorry I cannot agree with this. `InstanceData` is a way to efficiently add a new field in a runtime per each task. Do you have any idea instead?

Reply:
I think our data entity is enough to store data or results.
If you need a temporary data structure in the meantime, I think a dictionary can be a good choice because in `PostDataSample` only `set_field` is used for setter.

Reply:
Let's discuss this in the near future.